index.html

```diff
@@ -68,7 +68,7 @@ <h3>See also...</h3>
 <ul>
 <li>Frederic Koehler's <a href="https://github.com/frytvm/XS">C++ version, with language changes</a></li>
 <li>Olaf Rogalsky's <a href="http://web.archive.org/web/20070626160532/http://theorie1.physik.uni-erlangen.de/rogalsky/es/es.html">job control version</a></li>
-<li>Kevin Lipe's <a href="https://github.com/epilnivek/kes">OSX changes</a></li>
+<li>Kevin Lipe's <a href="https://web.archive.org/web/20160719041044/github.com/epilnivek/kes">OSX changes</a></li>
```

Comment:
I think it would be better to use Software Heritage:
```suggestion
<li>Kevin Lipe's <a href="https://archive.softwareheritage.org/browse/origin/https://github.com/epilnivek/kes/directory/">OSX changes</a></li>
```